### PR TITLE
Fix create room icon color in dark mode

### PR DESCRIPTION
**Pull request by jrafaelsantana**
This patch fixes the messenger group icon color on dark theme.

Before:
image

After:
image

### CODE REVIEW

**Review by dusansimic — CHANGES_REQUESTED** [review]
Please add a new line to the end of the file. Everything else looks perfect 😁.

File: css/dark-mode.css [hunk 1]
```diff
@@ -217,3 +217,8 @@ html.dark-mode .x1ybostu {
 html.dark-mode .x4noic3 {
 	background-color: var(--secondary-button-background);
 }
+
+/* Messenger Group icon */
```
Comment (dusansimic) [inline css/dark-mode.css:221]:
```suggestion
/* Create room icon color */
```

**Review by dusansimic — CHANGES_REQUESTED** [review]

**Comment by jrafaelsantana** [comment]
Done!
Could you tag this PR with the hacktoberfest-accepted label?

Thank you!

**Review by dusansimic — APPROVED** [review]

**Comment by dusansimic** [comment]
> Could you tag this PR with the hacktoberfest-accepted label?

Of course! Just wanted to make sure everything is in order. I'll merge it when CI passes.

**Comment by dusansimic** [comment]
@jrafaelsantana could your resolve the conflict?

**Comment by dusansimic** [comment]
Thanks for the fix!

**Pull request merged**